subprojects/testfx-core/src/test/java/org/testfx/matcher/control/ComboBoxMatchersTest.java

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright 2013-2014 SmartBear Software
+ * Copyright 2014-2015 The TestFX Contributors
+ *
+ * Licensed under the EUPL, Version 1.1 or - as soon they will be approved by the
+ * European Commission - subsequent versions of the EUPL (the "Licence"); You may
+ * not use this work except in compliance with the Licence.
+ *
+ * You may obtain a copy of the Licence at:
+ * http://ec.europa.eu/idabc/eupl
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the Licence is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the Licence for the
+ * specific language governing permissions and limitations under the Licence.
+ */
+package org.testfx.matcher.control;
+
+import javafx.scene.control.ComboBox;
+import javafx.scene.input.KeyCode;
+import javafx.scene.layout.StackPane;
+
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.testfx.api.FxRobot;
+import org.testfx.api.FxToolkit;
+
+import static javafx.collections.FXCollections.observableArrayList;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class ComboBoxMatchersTest extends FxRobot {
+
+    //---------------------------------------------------------------------------------------------
+    // FIELDS.
+    //---------------------------------------------------------------------------------------------
+
+    @Rule
+    public ExpectedException exception = ExpectedException.none();
+
+    public ComboBox<String> comboBox;
+
+    //---------------------------------------------------------------------------------------------
+    // FIXTURE METHODS.
+    //---------------------------------------------------------------------------------------------
+
+    @BeforeClass
+    public static void setupSpec() throws Exception {
+        FxToolkit.registerPrimaryStage();
+    }
+
+    @Before
+    public void setup() throws Exception {
+        FxToolkit.setupSceneRoot(() -> {
+            comboBox = new ComboBox<>();
+            comboBox.setItems(observableArrayList("alice", "bob", "carol", "dave"));
+            comboBox.getSelectionModel().selectFirst();
```

Comment:
This is okay. Alternatively we could use a mouse robot method here. But to use `selectFirst()` directly seems to be better in this case.